src/content/en/2021/caching.md

```diff
@@ -17,4 +19,328 @@ featured_stat_3: TODO
+
+## Headers adoption
+
+With both CDN and Browser, the HTTP Headers are the leading toolkit each developer needs to master to properly cache resources. Headers are simply instructions known as "directives", read during the HTTP Request or Response to control the cache strategy.
+
+The caching-related headers, or the absence of them, tell the browser or CDN three essential pieces of information:
+- **Cacheability**: Is this content cacheable?
+- **Freshness**: If it is cacheable, how long can it be cached for? `Cache-Control` and `Expires` 
+- **Validation**: If it is cacheable, how do I ensure that my cached version is still fresh?
+
+Headers are meant to be used either alone or together. To tell if the content is cachable and fresh, we have:
+- `Expires` specifies an explicit expiration date and time (i.e., when precisely the content expires)
+- `Cache-Control` specifies a cache duration (i.e., how long the content can be cached in the browser relative to when it was requested)
+In case both are specified, `Cache-Control` takes precedence.
+
+The usage of the Cache-Control header has increased steadily since 2019. 74% of responses on Mobile pages included the Cache-Control header, while 75% of responses on Desktop pages utilized the header. From 2020, the usage of this specific header increased by 0.71% for Mobile and by 1.13% for Desktop.
+
+{{ figure_markup(
+  image="cache-control-expires.png",
+  caption="Percent of responses that set `Cache-Control` and `Expires` headers.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=816405026&format=interactive",
+  sheets_gid="2102749619",
+  sql_file="header_trends.sql"
+) }}
+
+To validate the content, we have:
+- `Last-Modified` indicates when the object was last changed. Its value is a date timestamp.
+- `ETag` (Entity Tag) provides a unique identifier for the content as a quoted string. It can take any format the server chooses; it is typically a hash of the file contents, but it could be a timestamp or a simple string.
+
+In case both are specified, `ETag` takes precedence.
+
+{{ figure_markup(
+  image="last-modified-etag.png",
+  caption="Percent of responses that set `Last-Modified` and `ETag` headers.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=1490687174&format=interactive",
+  sheets_gid="2102749619",
+  sql_file="header_trends.sql"
+) }}
+
+### Cachebility and freshness
+The `Expires` header, as stated above, it indicates the exact date/time after which the response is considered stale. Therefore, its value is a date and time, such as:
```

Comment:
Edits: 

The `Expires` header, as stated above, indicates the exact date/time after which the response is considered stale. Therefore, its value is a date and time, such as:

```
Expires: Mon, 29 Nov 2021 16:00:00 GMT
```

This was perfect for the early days of HTTP/1.0, and helped Browsers understand how long they could cache a resource.

HTTP/1.1 took a step further by introducing the Cache-Control header, supported by all commonly used browsers for a long time. The Cache-Control header provides much more extensibility and flexibility than Expires via caching directives, several of which can be specified together, making it a better fit in many cases.

Request:
```
> GET /static/js/main.js HTTP/2
> Host: www.example.org
> Accept: */*
```

Response:
```
< HTTP/2 200
< Date: Mon, 29 Nov 2021 08:04:17 GMT
< Expires: Mon, 29 Nov 2021 08:34:17 GMT
< Cache-Control: public, max-age=1200
```

The simple example above shows a request and response for a JavaScript file. First, the `Date` header indicates the current date (precisely when the content was served). Next, the `Expires` header indicates that it can be cached for 30 minutes (the difference between the `Expires` and `Date` headers). Finally, the `Cache-Control` header specifies the `max-age` directive, which indicates that the resource can be cached for 600 seconds (10 minutes). Since `Cache-Control` takes precedence over `Expires`, the browser will cache the response for 10 minutes, after which it will be marked as stale.

It's worth mentioning that if no caching headers are present in a response, then the browser is allowed to *heuristically* cache the response. Some may cache the response indefinitely, and some may not cache it at all. Because of this variance between browsers, it is essential to explicitly set specific caching rules to ensure that you control the cacheability of your content.

Reply:
@jessnicolet @Zizzamia this seems to draw heavily from the [2020 chapter](https://almanac.httparchive.org/en/2020/caching#cache-control-vs-expires), even reusing the code examples and phrases.

We've seen this kind of reuse in a few other chapters and we've been encouraging authors to try to be more original for the sake of giving readers a fresh take on the subject.

Beyond that I'd also advise against too much of a history lesson or how-to. Rather, as I wrote in one of the recent [weekly roundups](https://github.com/orgs/HTTPArchive/teams/2021-web-almanac/discussions/11), we want the narrative to be data-driven:

> Remember that our focus is to report on the state of the web. There are tons of resources that teach developers how to use web technologies, but our content is unique in that we have access to a wealth of data about how those technologies are used in the wild. So even though many of us are great educators and can lecture about the history and proper implementation of some specific topic, think of your chapter like a research paper. We've got a ton of stats and we're excited to share them with the web community. That doesn't mean that we can't explain a concept before talking about its usage—we should still make the content approachable for beginners—but keep the narrative data-driven.

Reply:
Good point! Yeah for sake of time we are re-using some of the old template as placeholder, and then spin more original take. We are going to to cut some of the how-to so we keep it more data-driven. Thank you @rviscomi

Reply:
I guess let me cut this part entirely and stay more data-driven.